mypy check

Solve mypy check error:

elena/__init__.py:5: error: Name "version" already defined (possibly by an import)
Found 1 error in 1 file (checked 68 source files)

--- a/elena/__init__.py
+++ b/elena/__init__.py
@@ -1,8 +1,4 @@
-try:
-    from importlib.metadata import version
-except ImportError:
-    # For Python versions prior to 3.8
-    from importlib_metadata import version
+from importlib.metadata import version
 
 try:
     __version__ = version("elena")

--- a/elena/entrypoint/local/entrypoint.py
+++ b/elena/entrypoint/local/entrypoint.py
@@ -1,5 +1,10 @@
 from elena.domain.services.elena import get_elena_instance
 
-if __name__ == "__main__":
+
+def main():
     elena = get_elena_instance()
     elena.run()
+
+
+if __name__ == "__main__":
+    main()

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,6 +1,6 @@
 [metadata]
 name = elena
-version = 2.1.6
+version = 2.1.7
 author = "L Software Foundation"
 author_email = "fransimo@gmail.com, pjover@gmail.com"
 description = Hexagonal multi exchange crypto currency strategy executor

